### PR TITLE
Implemented a sample product page

### PR DESCRIPTION
**Pull request by GeronimoJohn**
Created new files called productPage.html and product.css. 
Inside the HTML file is a basic layout for selling products. 

Created a new folder for images that are going to be used in the future.

### CODE REVIEW

File: src/productPage.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- arkjhkjhfksjdhfkj -->
```
Comment (thomastayler) [inline src/productPage.html]: Is this required?

Comment (thomastayler) [inline src/productPage.html]: is this required?

File: src/productPage.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+            </div>
+        </div>
+    </nav>
+    <!-- Products -->
+    <div class="container">
+        <form class="form-inline my-2 my-md-0" method="POST" action="products.php">
+            <input class="form-control" type="text" placeholder="Search" aria-label="Search">
+            <button class="btn btn-warning" name="submit" id="show" onclick="getData()">Click</button>
+
+            <!-- TODO: bring out the data from the database -->
```
Comment (thomastayler) [inline src/productPage.html]: Could this be removed?

File: src/productPage.html (hunk 1 above)
Comment (thomastayler) [inline src/productPage.html]: is this required?

File: src/productPage.html [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                        </div>
+                        <a class="add-to-cart" href="">+ Add To Cart</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (thomastayler) [inline src/productPage.html]: no new line

File: src/productPage.html (hunk 1 above)
Comment (thomastayler) [inline src/productPage.html]: is this required?

File: src/productPage.html (hunk 3 above)
Comment (thomastayler) [inline src/productPage.html]: No new line at the end of code

**Review by thomastayler — CHANGES_REQUESTED** [review]

File: src/productPage.html (hunk 1 above)
Comment (thomastayler) [reply src/productPage.html]: Referencing to line 5

File: src/productPage.html (hunk 3 above)
Comment (GeronimoJohn) [reply src/productPage.html]: Will fix that

File: src/productPage.html (hunk 1 above)
Comment (GeronimoJohn) [reply src/productPage.html]: fixed requested changes

File: src/productPage.html (hunk 3 above)
Comment (thomastayler) [reply src/productPage.html]: in this case, it needs to be after line  the 'html' closing tag

**Review by thomastayler — APPROVED** [review]

**Pull request merged**